src/ignitus-Contributors/reducers.js

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
Could you check why we have this reducer twice or more? 
And also as the name suggest TestimonialReducer  it should only be in Testimonial component as per current coding structure.

Reply:
@outsourcegb Intially we have added boilerplate testimonial reducer, selecters etc everywhere but now everything is resolved as of now. you can check the updated codebase 👍

Reply:
@divyanshu-rawat  In that case, all looks good.